### ISSUE TITLE
New features

### ISSUE THREAD

**Issue by jwang917** [robertisandor/USF-MSDS621-KaggleHomeDepot#5]
Clean up the description, query, attributes by separating connected words and some other issues.

Add new similarity features: 
1. Jaro distance.
2. Normalized compression distance.
3. Smith-Waterman algorithm to compute similarity.

If time allowed, I will run PCA on our features and try on using SVM to do regression. 

Divide all features into three groups. 
Run SVM, XGBoost on similarity features. 
Plot model performances on feature space. 
Add PCA before running on all numerical features.
Investigate NCD, Jaro, Smith-waterman algorithm, and write them in plain English.

**Comment by jwang917** [comment]
Done with Jaro score computing.

**Comment by jwang917** [comment]
Done with Smith-Waterman similarity score

**Comment by jwang917** [comment]
Done with Normalized Compression Distance.

**Comment by jwang917** [comment]
Gonna do the PCA and SVM regression after afternoon class. And make a small data exploration on the features done in the last weekend

**Comment by jwang917** [comment]
Done with the dividing of features. Working on getting result for svm regression.

**Comment by robertisandor** [comment]
Features have been tested and integrated into the code. Thanks for your help!
